src/Sylius/Bundle/ApiBundle/Resources/config/services/filters.xml

```diff
@@ -359,14 +359,14 @@
             <tag name="api_platform.filter" />
         </service>
 
-        <service id="sylius.api.search_filter.product.code" parent="api_platform.doctrine.orm.search_filter" public="true">
+        <service id="sylius_api.search_filter.admin.product_taxon.product_code" parent="api_platform.doctrine.orm.search_filter" public="true">
             <argument type="collection">
                 <argument key="product.code">exact</argument>
             </argument>
             <tag name="api_platform.filter" />
         </service>
 
-        <service id="sylius.api.search_filter.taxon.code" parent="api_platform.doctrine.orm.search_filter" public="true">
+        <service id="sylius_api.search_filter.admin.product_taxon.taxon_code" parent="api_platform.doctrine.orm.search_filter" public="true">
```

Comment:
They should be merged into one `sylius_api.search_filter.admin.product_taxon` filter